pymbolic/primitives.py

```diff
@@ -270,16 +301,16 @@ class Expression:
         `PEP 634 <https://peps.python.org/pep-0634/>`__-style pattern matching
         is now supported when Pymbolic is used under Python 3.10.
 
-    .. attribute:: a
+    .. autoattribute:: a
 
-    .. attribute:: attr
+    .. autoattribute:: attr
```

Comment:
```suggestion
    .. automethod:: attr
```
? Not sure how this worked before..

Reply:
Done.